tests/integration/bond_test.py

```diff
@@ -255,9 +255,26 @@ def test_set_bond_mac_address(eth1_up):
         bond99_cur_state = current_state[INTERFACES][0]
         assert bond99_cur_state[Interface.MAC] == MAC0.upper()
 
+        bond99_slaves_mac, eth1_slaves_mac, _ = return_bond_and_slaves_mac()
+        assert bond99_slaves_mac == eth1_slaves_mac
+
         bond_state[Interface.KEY][0][Interface.MAC] = MAC1
         libnmstate.apply(bond_state)
 
         current_state = statelib.show_only(('bond99',))
         bond99_cur_state = current_state[INTERFACES][0]
         assert bond99_cur_state[Interface.MAC] == MAC1.upper()
+
+        bond99_slaves_mac, eth1_slaves_mac, _ = return_bond_and_slaves_mac()
+        assert bond99_slaves_mac == eth1_slaves_mac
+
+
+def return_bond_and_slaves_mac():
+    bond_and_slaves_state = statelib.show_only(('bond99', 'eth1', 'eth2'))
```

Comment:
The show needs to happen outside this function and the function should get a parameter like `state` and it could then just return the mac addresses of all interfaces in the state. Then it could be `get_macs()`.